builder/Dockerfile

```diff
@@ -72,8 +77,8 @@ RUN ssh-keygen -A \
   && mkdir /run/sshd
 
 # Add remote development user
-RUN useradd -m remoteuser \
-  && yes c0llectah | passwd remoteuser
+#RUN useradd -m remoteuser \
+#  && yes c0llectah | passwd remoteuser
```

Comment:
I commented out these commands because this was failing in Openshift CI. I don't know why. 

STEP 18/22: RUN useradd -m remoteuser   && yes c0llectah | passwd remoteuser
New password: Retype new password: Changing password for user remoteuser.
passwd: all authentication tokens updated successfully.
error: build error: error building at STEP "RUN useradd -m...d remoteuser": error while running runtime: exit status 141

Reply:
Looks like the issue with `yes` writing to a non existing process (when `passwd` has already finished), getting `SIGPIPE`. Would it work this way `(yes c0llectah || true) | passwd` ?